release-notes/opensearch-security-dashboards-plugin.release-notes-1.3.0.0.md

```diff
@@ -0,0 +1,24 @@
+## Version 1.3.0.0
+
+### Enhancements
+* Add auto backport functionality to security plugin ([#887](https://github.com/opensearch-project/security-dashboards-plugin/pull/887))
+* Adds auto delete workflow for backport branches ([#901](https://github.com/opensearch-project/security-dashboards-plugin/pull/901))
+* Configure ML plugin actions ([#912](https://github.com/opensearch-project/security-dashboards-plugin/pull/912))
+
+### Bug Fixes
+* Updates rule def for @osd/eslint/require-license-header ([#905](https://github.com/opensearch-project/security-dashboards-plugin/pull/905))
+* Updates backport workflow with custom branch and github app ([#900](https://github.com/opensearch-project/security-dashboards-plugin/pull/900))
+
+### Maintenance
+* Bumps version to 1.3.0.0 ([#884](https://github.com/opensearch-project/security-dashboards-plugin/pull/884))
+* Adds support for codeowners to repo ([#883](https://github.com/opensearch-project/security-dashboards-plugin/pull/883))
+* Adds .whitesource and configs file to activate whitesource integration ([#885](https://github.com/opensearch-project/security-dashboards-plugin/pull/885))
+* Uses 1.x branch of Dashboards for unit tests ([#890](https://github.com/opensearch-project/security-dashboards-plugin/pull/890))
+* Makes PR template easier to fill in ([#888](https://github.com/opensearch-project/security-dashboards-plugin/pull/888))
+* Adds release notes for 1.3.0.0 ([#918](https://github.com/opensearch-project/security-dashboards-plugin/pull/918))
+* Updates release notes for 1.3.0.0 ([#920](https://github.com/opensearch-project/security-dashboards-plugin/pull/920))
```

Comment:
We need to backport this PR separately.

Reply:
Ok. Can you explain a little more what needs to be done?